css/_toastr.scss

```diff
@@ -93,7 +93,7 @@
 
 #toast-container.notification-bottom-right {
   bottom: 135px;
-  right: 13px;
+  right: 28px;
 }
```

Comment:
We can improve this in a future PR, by making sure that this is the $hideFilmstripButtonWidth + 2*$hideFilmstripMargin. This way if we want to adjust the button, change the icon, etc. we don't have to care about fixing this.